libexec/cli/start.help

```diff
@@ -0,0 +1,79 @@
+USAGE: singularity [...] start [start options...] <container path> <instance name>
+
+Start a named instance of the given container image.
+
+
+START OPTIONS:
+    -B/--bind <spec>    A user-bind path specification.  spec has the format
+                        src[:dest[:opts]], where src and dest are outside and
+                        inside paths.  If dest is not given, it is set equal
+                        to src.  Mount options ('opts') may be specified as
+                        'ro' (read-only) or 'rw' (read/write, which is the
+                        default). This option can be called multiple times.
+    -c/--contain        This option disables the sharing of filesystems on 
+                        your host (e.g. /dev, $HOME and /tmp).
+    -e/--cleanenv       Clean environment before running container
+    -H/--home <spec>    A home directory specification.  spec can either be a
+                        src path or src:dest pair.  src is the source path
+                        of the home directory outside the container and dest
+                        overrides the home directory within the container
+    -n/--nv             Enable experimental Nvidia support
+    --pwd               Initial working directory for payload process inside
+                        the container
+    -S/--scratch <path> Include a scratch directory within the container that
+                        is linked to a temporary dir (use -W to force location)
+    -s/--shell <shell>  Path to program to use for interactive shell
```

Comment:
Does `--shell` make sense in the context of start?  Should it be removed from help?